### PR TITLE
Update cardano configs

### PR DESCRIPTION
**Pull request by erikd**

### CODE REVIEW

**Review by newhoggy — APPROVED** [review]

**Comment by erikd** [comment]
bors merge

**Comment by erikd** [comment]
bors merge

File: configuration/cardano/mainnet-config.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
   "TracingVerbosity": "NormalVerbosity",
   "TurnOnLogMetrics": true,
   "TurnOnLogging": true,
+  "ViewMode": "SimpleView",
```
Comment (mrBliss) [inline configuration/cardano/mainnet-config.json:44]: I thought the `LiveView` had been removed and there's no reason anymore to specify the `ViewMode`? Similarly for the other changes. cc: @denisshevchenko

Comment (erikd) [reply configuration/cardano/mainnet-config.json:44]: This was me trying to sync config with what is provided by CI, but it seems I synced in the wrong direction.

**Pull request merged**